cypress/e2e/registration.cy.js

```diff
@@ -2,10 +2,35 @@
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.viewport(1920, 1080);
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
 
-  it('', () => {
+  it('should provide an ability to register with valid creds', () => {
+    cy.get('#firstName').type('Alex');
```

Comment:
If you have value that appears more than once, you can create a constant with it and use it everywhere, for example const name = 'Alex';